src/server/handlers/static.go

```diff
@@ -39,6 +41,16 @@ func (s *Static) SetNext(h http.Handler) {
 func (s Static) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	path := s.MediaPath + r.URL.EscapedPath()
 
+	// Check for ".." in the url path,
+	// if we find ".." in the path we will not serve static files
+	doubleDotRegex := regexp.MustCompile("[.]{2,}")
+	if doubleDotRegex.MatchString(path) {
```

Comment:
my only concern with using regexp here is that you're introducing a new attack. Knowing that you're `regexp`-ing the path, someone may add a long path to make the server slow. This makes the server vuln. to DDoS IMHO.

Maybe it's good idea to bench test `strings.Contains` vs `Regexp` like [here](https://gist.github.com/smagch/9210646)

Reply:
Considering it doesn't seem like an 'Evil Regex', mainly because there is no use of * or + quantifiers, it *should* not be a real problem. That being said, it would probably be better to use strings.Contains, to avoid unnecessary overhead.

Reply:
:+1:  changed to `strings.Contains`